index.html

```diff
@@ -1,24 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./index.css" />
-    <title>Awesome Books</title>
-  </head>
-  <body>
-    <header>
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://cdn.jsdelivr.net/npm/luxon@2.0.2/build/global/luxon.min.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="index.css">
+ <title> Awesome Books</title>
+</head>
+<body>
+  <main class="main-container">
+    <div class="menu">
+      <nav class="navbar-container">
+        <a class="logo"><span>Awesome Books</span></a>
+        <ul class="nav-list">
+          <li id="add-list" class="nav-link">List</li>
+          <li id="add-new" class="nav-link">Add New</li>
+          <li id="contact" class="nav-link">Contact</li>
+        </ul>
+        <div class="hamburger">
+          <div class="line1"></div>
+          <div class="line2"></div>
+          <div class="line3"></div>
+        </div>
+      </nav>
+    </div>
+    <div class="set-time">
+      <p class="today-date"></p>
+    </div>
+    <div class="awesome">
       <h1>Awesome Books</h1>
-    </header>
-    <main id="main">
       <div class="books"></div>
+    </div>
+    <section class="input-data">
+      <h3>Add A Book</h3>
       <form method="get">
         <input id="title" type="text" placeholder="Title" required><br><br>
         <input id="author" type="text" placeholder="Author" required><br><br>
         <button type="submit" id="add">Add</button>
       </form>
```

Comment:
- [ ] Please make sure to follow the design of the input field and the add button. Please fix that.

### Your design

![Screenshot from 2021-11-25 19-51-28](https://user-images.githubusercontent.com/22434660/143458978-84ba0b98-7400-48e1-a985-129b387345ca.png)


### Original desig

![Screenshot from 2021-11-25 19-53-23](https://user-images.githubusercontent.com/22434660/143459021-324bc095-f384-437b-96e5-033af8da4951.png)
n